content/channels/index.textile

```diff
@@ -29,6 +29,41 @@ The following are examples of channels that are all part of the 'customer' names
 * customer:tracking-id
 * customer:order:update
 
+
+<aside data-type='further-reading'>
+  <h3> This is further reading </h3>
+  <ul>
+    <li> scope </li>
```

Comment:
Do we need to remove this? or Mark will update this file later?

Reply:
Mark will update :)

Reply:
Can we remove these please to keep the content history clean?